sdk/src/test/java/io/customer/sdk/repositories/AttributesRepositoryTest.kt

```diff
@@ -0,0 +1,108 @@
+package io.customer.sdk.repositories
+
+import io.customer.base.extenstions.getUnixTimestamp
+import io.customer.sdk.data.moshi.CustomerIOParser
+import io.customer.sdk.data.moshi.CustomerIOParserImpl
+import io.customer.sdk.repository.AttributesRepository
+import io.customer.sdk.repository.MoshiAttributesRepositoryImp
+import org.amshove.kluent.shouldBeTrue
+import org.junit.Before
+import org.junit.Test
+import java.util.*
+
+internal class AttributesRepositoryTest {
+
+    private val parser: CustomerIOParser = CustomerIOParserImpl()
+    lateinit var attributesRepository: AttributesRepository
+
+    @Before
+    fun setup() {
+        attributesRepository = MoshiAttributesRepositoryImp(parser)
+    }
+
+    @Test
+    fun `Verify string attributes are mapped correctly`() {
+        val expected = mapOf("key" to "pair", "key2" to "pair2")
+
+        val result = attributesRepository.mapToJson(mapOf("key" to "pair", "key2" to "pair2"))
+
+        (result == expected).shouldBeTrue()
```

Comment:
Can we write this as `result shouldBeEqual expected`? Using *should equal* over *should true*?

Reply:
`shouldBeEqual` in case of map, matches the order as well. But we are not using ordered maps, so it would fail unless we change all the maps to ordered maps.